src/main/java/mate/academy/springboot/swagger/model/Product.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.swagger.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "products")
+public class Product {
```

Comment:
missing equals, hashcode and toString

Reply:
what sense for generate equals, hashcode and toString?

Reply:
this is a model class that can be stored in all sorts of data structures, and some of them require an overridden hashcode, equals is just a must-have in any situation, to string is just handy in debugging

Also, it's a bit late to ask those type of questions, on such an important thing